Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -7,6 +7,9 @@
 #:     If `--new-issue` is passed, automatically create a new issue in the appropriate
 #:     GitHub repository as well as creating the Gist.
 #:
+#:     If `--test-logs` is passed, uploads the logs for the formula's last test run, instead
+#:     of the build logs.
```

Comment:
Yeah. After seeing other folks' feedback about alternate dir organization, I think "just always upload everything" is the right behavior, too. Going to remove this option and fix `gist-logs` to include the logs from all steps, with some rename logic to make it sensible.